package.json

```diff
@@ -2,15 +2,15 @@
   "name": "madison",
   "description": "A dirt simple utility for working with US state names and abbreviations.",
   "author": "Mike Ball <mikedball@gmail.com>",
-  "version": "0.0.9",
+  "version": "1.0.0",
```

Comment:
Well, I am normally operating under the assumption that the first release of the "public API" always starts at the version 1.0.0. I'm referencing bullet 4 from [semver.org](http://semver.org/).